bobcat/bb-cumber/src/main/java/com/cognifide/qa/bb/cumber/BobcumberListener.java

```diff
@@ -96,4 +97,17 @@ private synchronized void addScenario(String failedScenario) throws IOException
       featureMap.writeFeatures(out);
     }
   }
+
+  private boolean isScenario(String displayName) {
+    boolean isScenario = false;
+    if (displayName.contains(COLON)) {
+      String testStep = displayName.substring(0, displayName.lastIndexOf(COLON));
```

Comment:
Or `String testStep = StringUtils.substringBeforeLast(displayName, COLON);`
Same goes for other operations on strings, `StringUtils` operations are null-safe (just checked the Cucumber implementation, it does return null for displayName but who knows what happens in the future.